masterlist.yaml

```
@@ -6383,6 +6383,31 @@ plugins:
         itm: 8
         udr: 105
 
+  - name: 'Qarls_Harvest addon(_MMM)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/8285/'
+        name: 'Qarls Harvest Addon'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Qarls Harvest Addon ESP' ]
+        condition: 'many("Qarls_Harvest addon(_MMM)?\.esp")'
+  - name: 'Qarls_Harvest addon.esp'
+    msg:
+      - <<: *useVersion
+        subs: [ 'Qarls_Harvest addon_MMM.esp' ]
+        condition: 'active("Mart''s Monster Mod.esm")'
```

For useVersion I would specify the mod name, useInstead is better if you want to list a specific plugin name.
```suggestion
      - <<: *useVersion
        subs: [ 'Mart''s Monster Mod' ]
        condition: 'active("Mart''s Monster Mod.esm")'
```